### PR TITLE
GP2-2095 Error page styling

### PR DESCRIPTION
**Pull request by bobmeredith**
Changes the styling and content on the 404 page and adds a couple of links
I took the opportunity to rationalise the use of the templatetag 'url_map' throughout the site too. It now does a lookup from the slugs file - so we only have one place to define wagtail specific URLs.

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/GP2-2095
- [x] Jira ticket has the correct status.
- [x] [Changelog](CHANGELOG.md) entry added.

### Reviewing help

- [x] Explains how to test locally, including how to set up appropriate data
- [x] Includes screenshot(s) - ideally before and after, but at least after

### Housekeeping

- [ ] Added all new environment variables to Vault.
- [ ] Cleaned up old feature flags
- [ ] Upgraded any vulnerable dependencies.
- [ ] I have updated security dependencies
- [ ] Python requirements have been re-compiled.
- [ ] Frontend assets have been re-compiled.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

**Comment by bobmeredith** [comment]
# Pinned because pre-commit features specific versions
#black==20.8b1

**Comment by bobmeredith** [comment]
image

**Comment by bobmeredith** [comment]
Testing: - type some non-existent page on the URL and bingo!
Chanck links are valid

File: core/templates/core/base.html [hunk 1]
```diff
@@ -101,7 +101,7 @@
       magna.setInitialState({
         // prevents modals from opening on page load if user dismissed the modal already
         performSkipFeatureCookieCheck: true,
-        nextUrl: '{% if request.GET.next %}{{ request.GET.next }}{% else %}{% url_map "dashboard" %}{% endif %}',
+        nextUrl: '{% if request.GET.next %}{{ request.GET.next }}{% else %}{% url_map "DASHBOARD_URL" %}{% endif %}',
```
Comment (rafa-garcia) [inline core/templates/core/base.html:104]: Any reason to swap the case of `dashboard_url` across templates?

Comment (bobmeredith) [reply core/templates/core/base.html:104]: No - actually, I just left the case as it was.  They should all be consistent really.

**Review by rafa-garcia — APPROVED** [review]
LGTM

**Pull request merged**